helm/kong-app/templates/_helpers.tpl

```diff
@@ -190,7 +190,7 @@ The name of the service used for the ingress controller's validation webhook
 {{- $_ := set $autoEnv "CONTROLLER_ELECTION_ID" (printf "kong-ingress-controller-leader-%s" .Values.ingressController.ingressClass) -}}
 {{- $_ := set $autoEnv "CONTROLLER_KONG_URL" (include "kong.adminLocalURL" .) -}}
 {{- if .Values.ingressController.admissionWebhook.enabled }}
-  {{- $_ := set $autoEnv "CONTROLLER_ADMISSION_WEBHOOK_LISTEN" (printf "0.0.0.0:%d" .Values.ingressController.admissionWebhook.port) -}}
+  {{- $_ := set $autoEnv "CONTROLLER_ADMISSION_WEBHOOK_LISTEN" (printf "0.0.0.0:%s" .Values.ingressController.admissionWebhook.port) -}}
```

Comment:
Issue is present in upstream chart https://github.com/Kong/charts/blob/master/charts/kong/templates/_helpers.tpl#L193

Yasser was putting lots of effort to minimize customizations inventory, try to push fixes like this to the upstream (eventually?). He was mentioning having some automation to sync with the upstream changes, while still making it possible to have some customizations in our "fork". Not sure about the process details but curious to learn.

Will check readme if process was documented, or can derive it from history. @ced0ps do you maybe know already how Yasser was doing it? 

//cc @yasn77

Reply:
ok let's unblock our customer and I do a PR to upstream now

Reply:
Yep... @pipo02mix could you also create a PR upstream? 

The way I am keeping things in sync is using https://gist.github.com/yasn77/cb0600bebb9237fbd73d7ca67b4ff211. That downloads the tarballs and then does a cherry-pick of commits. But if the change already exists in this fork, then it's just ignored. So it's fine to make a change here first (so out customers are ok) and later pull the change from upstream.